quiche/Cargo.toml

```diff
@@ -57,6 +57,7 @@ features = ["boringssl-boring-crate", "qlog"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [build-dependencies]
+cdylib-link-lines = "0.1"
```

Comment:
I think we probably want to limit this dependency to when the `ffi` feature is enabled, it's not very useful otherwise (we do still build the library in that case, but there's no way around that IIRC).

Reply:
Ok. I updated that it depends on `ffi` feature.